src/ashley/views.py

```diff
@@ -4,18 +4,30 @@
 from typing import List
 
 from django.conf import settings
+from django.contrib import messages
+from django.contrib.auth import get_user_model
 from django.http import HttpResponse, HttpResponseBadRequest, HttpResponseRedirect
 from django.urls import reverse
 from django.utils import translation
+from django.utils.translation import gettext_lazy as _
+from django.views.generic import UpdateView
 from lti_toolbox.lti import LTI
 from lti_toolbox.views import BaseLTIAuthView
 from machina.apps.forum_permission.shortcuts import assign_perm
+from machina.apps.forum_permission.viewmixins import (
+    PermissionRequiredMixin as BasePermissionRequiredMixin,
```

Comment:
I would have added tests on this view, especially on permissions.

Reply:
So this is what I forgot! :sweat_smile: 
I added tests for this view, thanks!